_redirects

```diff
@@ -41,6 +41,7 @@
 /docs/api														/docs/concepts/overview/kubernetes-api		301
 
 /docs/concepts/abstractions/controllers/garbage-collection		/docs/concepts/workloads/controllers/garbage-collection		301
+/docs/concepts/abstractions/controllers/petsets							301
```

Comment:
@steveperry-53 The reason I initially removed all `petset` links as `petset` is officially retired for 1.8 when it rolls out. Based on your feedback, I retained all links at left side. As `petset` is retired, there should not be any link to its documentation, so i still removed all `petset` links at right side. Is it correct?

Should i also remove `301` if all the `petset` links are removed at right side?